data/cuisines.txt

```diff
@@ -161,35 +161,36 @@
 
   [coffee_shop]
     tags = android, ios
-    en = Coffee Shop
```

Comment:
Потому что программно делался тайтл кейз

> On 18.3.2016, at 15.43, Ilya Zverev <notifications@github.com> wrote:
> 
> In data/cuisines.txt:
> 
> > @@ -161,35 +161,36 @@
> >  
> >    [coffee_shop]
> >      tags = android, ios
> > -    en = Coffee Shop
> Интересно, а если раньше здесь было с заглавной буквы, то почему во всех готовых файлах было со строчной?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>